spec/services/publishing_api_payload_spec.rb

```diff
@@ -61,6 +61,33 @@
       expect(payload).to match a_hash_including(payload_hash)
     end
 
+    it "converts role appointment links to role and person links" do
+      role_appointment_id = SecureRandom.uuid
+      role_appointments_schema = build(:tag_schema, type: "multi_tag", id: "role_appointments")
+      document_type_schema = build(:document_type_schema, tags: [role_appointments_schema])
+      document = build(:document, document_type: document_type_schema.id, tags: {
+                         role_appointments: [role_appointment_id],
+                       })
+
+      person_id = SecureRandom.uuid
+      role_id = SecureRandom.uuid
+      links = {
+        "content_id" => role_appointment_id,
+        "links" => { "person" => [person_id], "role" => [role_id] },
+      }
+
+      # TODO: add this to gds-api-adapters test helpers
```

Comment:
Is this already there...

https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/test_helpers/publishing_api_v2.rb#L384

Reply:
d'oh! thanks.